serenity-core/src/main/java/net/thucydides/core/steps/PageObjectDependencyInjector.java

```diff
@@ -95,6 +95,7 @@ private boolean shouldInstrumentField(Field pageObjectField) {
 
     private void updatePageObject(Object pageObject) {//, Pages pages) {
         if (pageObject instanceof PageObject) {
+            ((PageObject) pageObject).setDriver(Serenity.getWebdriverManager().getWebdriver());
```

Comment:
This doesn't looks right. You are bypassing the hasAnInstantiatedDriver() test, which may cause problems elsewhere. If you think this is where the problem comes from, you may need to check in that method. Otherwise there may be cases where that method will return null.

Can you add a sample test that reproduces the issue (e.g. in WhenBrowsingAWebSiteUsingPageObjects) and proves that the fix works correctly?